packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx

```diff
@@ -216,6 +223,7 @@ const UserSettingsMenuBody = (props: MenuBodyProps) => {
         </a>
         <MenuItem hasDivider={true} />
         <a
```

Comment:
Btw since this is an `<a>` element, it's already selectable with tabs, so we could remove the `tabIndex={0}` on this line, right?  https://github.com/commercetools/merchant-center-application-kit/pull/1571/files#diff-8605567988365343f9b67c8b17dc2c7dL224

Just noticed this because when we try to navigate the menu items, the Logout item has two elements that get selected and the second one doesn't do anything when pressed and this seems to be the cause.

Reply:
Hm. Might be true would like to not mix this here though on the PR.